packages/forms/src/helpers.php

```diff
@@ -35,7 +35,7 @@ function array_move_before(array $array, $keyToMoveBefore): array
         $indexToMoveBefore = array_search($keyToMoveBefore, $keys);
         $keyToMoveAfter = $keys[$indexToMoveBefore - 1] ?? null;
 
-        if (filled($keyToMoveAfter)) {
```

Comment:
...and also this

Reply:
Ah, forget it. I thouht it would be more readable.